controllers/games.js

```diff
@@ -10,10 +12,29 @@ exports.getCategories = (req, res) => {
     })
 };
 
-exports.getReviews = (req, res) => {
+exports.getReviews = (req, res, next) => {
     let reviews = [];
     let count = 0;
-    selectReviews()
+
+    let category = null;
+    let sort_by = 'created_at';
+    let order = 'DESC';
+    
+    if(req.query.category){
+        category = req.query.category
+    };
+
+    if(req.query.sort_by){
+        sort_by = req.query.sort_by
+    };
+    
+    if(req.query.order){
+       if(req.query.order === 'ASC'){
+        order = 'ASC' 
+    };
+    };
```

Comment:
= you can do this logic in the model, with default values in the function parameters such as
(sort_by = created_at, category, order = 'DESC)